src/SFA.DAS.AANHub.Application/Models/AttendeeModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Models;
+
+public record AttendeeModel(Guid AttendanceId, Guid MemberId, string MemberName)
+{
+    public static implicit operator AttendeeModel(Attendance source) => new(source.Id, source.MemberId, source.Member.FullName);
```

Comment:
Can we have some coverate on this implicit operator? thanks

Reply:
Test coverage added